const get_req = async(req,res)=>{
    try {
        const req = fetch()
    } catch (error) {
        console.log(error)
    }
}